### DISCUSSION TITLE
Remove Quantity Button with Bundled Products

### DISCUSSION THREAD

**Discussion by jameshahn2** [woocommerce/woocommerce#45030] [WooCommerce, Blocks, and Site Editing]
### Discussed in https://github.com/woocommerce/woocommerce-gutenberg-products-block/discussions/5645

<div type='discussions-op-text'>

<sup>Originally posted by **jameshahn2** January 26, 2022</sup>
<p>I’ve been stuck for some time attempting to add a conditional for Product Bundles. It seems simple enough. Just trying to remove the quantity button on the cart page, but not for all products. Just for Product Bundles.</p>

Does anyone know what `if` I should add to this filter from the latest blog post?<br/>

``` 
add_filter( 'woocommerce_store_api_product_quantity_editable', function( $quantity, $product, $cart_item )  {
     return false;
}, 10, 3 );
```

I've tried every answer on Gist and Github and SO and I got nothing.<br/>

Any ideas?<br/>

Thank you!<br/>

James</div>

**Comment by manospsyx** [comment]
Hey @jameshahn2 👋 

[This](https://woocommerce.com/document/bundles/bundles-functions-reference/#section-2) document might come in handy!

In short:

- Use `wc_pb_is_bundled_cart_item( $cart_item )` to tell if a cart item is part of a bundle.
- Use `wc_pb_is_bundle_container_cart_item( $cart_item )` to tell if a cart item is a bundle container cart item.

Does this help?

**Reply by jameshahn2** [reply]
Wonderful! I’ve come back to this page several times and couldn’t quite connect those dots. Thank you SO much!
